src/core/operations/LuhnChecksum.mjs

```diff
@@ -0,0 +1,62 @@
+/**
+ * @author n1073645 [n1073645@gmail.com]
+ * @copyright Crown Copyright 2020
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation.mjs";
+import OperationError from "../errors/OperationError.mjs";
+
+/**
+ * Luhn Checksum operation
+ */
+class LuhnChecksum extends Operation {
+
+    /**
+     * LuhnChecksum constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "Luhn Checksum";
+        this.module = "Crypto";
```

Comment:
No need to add this to the crypto module since it has no imports. The modules are generally for operations that are likely to import large libraries. I'd just throw this into Default since it's so small. No need to update this PR though, I'll do it myself as I merge it.